tests/cmds/conftest.py

```diff
@@ -119,3 +117,9 @@ def gen(*args, **kwargs):
         yield Py42Response(response)
 
     return gen
+
+
+def get_mark_for_search_and_send_to(command_group):
```

Comment:
this method created parametized  tests specifically for both `search` and `send-to` and the testing diff looks complex because mostly started using this instead of the disparate other strategies that were employed